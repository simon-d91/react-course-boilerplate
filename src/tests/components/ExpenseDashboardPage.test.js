import React from 'react';
import { shallow } from 'enzyme';
import DasboardPage from '../../components/DashboardPage';

test('should render Dashboard page correctly', () => {
    const wrapper = shallow(<DasboardPage />);
    expect(wrapper).toMatchSnapshot();
})