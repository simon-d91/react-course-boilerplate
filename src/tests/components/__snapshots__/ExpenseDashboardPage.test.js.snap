// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Dashboard page correctly 1`] = `
<div>
  Dahboard Page content
</div>
`;
